src/constants/button.js

```diff
@@ -4,8 +4,15 @@ export const BUTTON_LABEL = {
     CONNECT: ('connect' : 'connect')
 };
 
+export const BUTTON_FUNDING_TYPE = {
+    CREDIT: ('credit' : 'credit'),
+    PAYPAL: ('paypal' : 'paypal')
+};
+
 export const BUTTON_COLOR = {
-    BLUE: ('blue' : 'blue')
+    GOLD:        ('gold' : 'gold'),
+    BLUE:        ('blue' : 'blue'),
+    DARKBLUE:    ('darkblue' : 'darkblue')
```

Comment:
Can we make sure that:

- Gold and blue are only usable by paypal button
- Dark blue is only usable by credit button

To make sure we're aligning with brand